### PR TITLE
19 02 remove lapack blas deps

### PR DESCRIPTION
**Pull request by hoytak**
Resolves https://github.com/apple/turicreate/issues/1419

### CODE REVIEW

File: src/unity/lib/gl_sframe.cpp [hunk 1]
```diff
@@ -266,9 +265,8 @@ std::vector<flexible_type> gl_sframe::operator[](int64_t i) {
   if (i < 0 || (size_t)i >= size()) {
     throw std::string("Index out of range");
   }
-  ensure_has_sframe_reader();
   std::vector<std::vector<flexible_type> > rows(1);
-  size_t rows_read  = m_sframe_reader->read_rows(i, i + 1, rows);
```
Comment (znation) [inline src/unity/lib/gl_sframe.cpp]: These changes look unrelated... should they be in a separate branch?

Comment (hoytak) [reply src/unity/lib/gl_sframe.cpp]: Ooops -- should not have pulled those in.  IDK
